pages/desktop/frontend/collections.py

```diff
@@ -118,6 +118,8 @@ class CollectionCreate(Region):
             '.EditableCollectionAddon',
         )
 
+        _warning_text_locator = (By.CSS_SELECTOR, '.Notice-error > * > * >.Notice-text')
```

Comment:
Could you not just use ```.Notice-text```?

Reply:
Just '.Notice-text' will find me the yellow notice too. 
I change it to '.Notice-error .Notice-text' :)